beams/app/model/fit.py

```diff
@@ -164,19 +178,61 @@ def __call__(self, *args, **kwargs):
                     try:
                         unnamed_pars.append(float(arg))
                     except ValueError:
-                        raise InvalidFitArgumentsError("Every parameter after the array should be of type FitParameter or a number.")
+                        raise InvalidFitArgumentsError(
+                            "Every parameter after the array should be of type FitParameter or a number.")
         for k, v in kwargs.items():
             if k != INDEPENDENT_VARIABLE:
                 pars[_replace_unsupported_unicode_characters(k)] = v
 
         try:
-            return self.__expression(time_array, *unnamed_pars, **pars).real
+            return self._expression(time_array, *unnamed_pars, **pars).real
         except TypeError as e:
             if ALPHA in pars.keys():
                 pars.pop(ALPHA)
-                return self.__expression(time_array, *unnamed_pars, **pars).real
+                return self._expression(time_array, *unnamed_pars, **pars).real
             raise InvalidFitArgumentsError("Bad arguments passed to fit expression.") from e
 
+    def __load_external_functions(self, external_function_dict):
+        loaded_external_functions = {}
+        for filename, full_file_path in external_function_dict.items():
+            spec = importlib.util.spec_from_file_location(filename, full_file_path)
+            module = importlib.util.module_from_spec(spec)
+            sys.modules[filename] = module
+            spec.loader.exec_module(module)
+            func = getattr(module, filename)
+            loaded_external_functions[filename] = func
+        return loaded_external_functions
+
+    def _alpha_correction(self, __expression):
+        def alpha_corrected_expression(time_array, *args, **kwargs):
+            args = list(args)
+            a = args.pop(-1)
+            exp_result = __expression(time_array, *args, **kwargs)
+            alpha_corrected = np.array(
+                [((1 - a) + ((1 + a) * point)) / ((1 + a) + ((1 - a) * point)) for point in exp_result])
```

Comment:
You can do 
```
exp_result = np.array(exp_result)
alpha_corrected = ((1 - a) + ((1 + a) * exp_result)) / ((1 + a) + ((1 - a) * exp_result))
```
which is more performant when we get into arrays larger then 20ish values. It's barely noticeable but the function gets called a lot so maybe?